geomesa-gs-plugin/geomesa-kafka-gs-plugin/README.md

```diff
@@ -9,3 +9,11 @@ You will also need to install the the following jars into geoserver/WEB-INF/lib
 * metrics-core-2.2.0.jar
 * zkclient-0.3.jar
 * zookeeper-3.4.5.jar
+
+For kafka 0.9.0.1 you will need these instead:
+* geomesa-kafka-09-utils-<version>.jar
+* kafka-clients-0.9.0.1.jar
+* kafka_2.11-0.9.0.1.jar
+* metrics-core-2.2.0.jar
+* zkclient-0.7.jar
+* zookeeper-3.4.5.jar
```

Comment:
@ddseapy can you update this script: https://github.com/ddseapy/geomesa/blob/kafka9final3/geomesa-tools/bin/install-kafka.sh

Maybe add another param (0.8 or 0.9) for the version number